### PR TITLE
Reviewed css class naming

### PR DESCRIPTION
**Pull request by ClementGautier**
| Q                 | A
| ----------------- | ---
| Specs             | N/A
| Behats            | N/A
| Blue CI           | Y
| Changelog updated | N/A

About
--------
I added a twig extension sooner (https://github.com/akeneo/pim-community-dev/pull/3850) but after a discussion with our UX, he said to me that we already have a css class that do the same thing as the one I added so I reviewed it here.

We discussed to add a no-margin class too as I need it in another context and he agreed.

### CODE REVIEW

File: features/Context/DataGridContext.php [hunk 1]
```diff
@@ -67,6 +67,60 @@ public function setPageFactory(PageFactory $pageFactory)
...
+     * @param string $item
+     * @param string $column
+     * @param string $datagrid
+     */
+    public function assertDatagridCellContainsText($text, $item, $column, $datagrid)
+    {
+        $gridName = $this->getGridName($datagrid);
+        $session = $this->getSession();
+
+        $positions = $this->spin(function () use ($gridName, $item, $column, $session) {
```
Comment (juliensnz) [inline features/Context/DataGridContext.php]: way to much stuff in this spin

File: features/Context/DataGridContext.php [hunk 2]
```diff
@@ -67,6 +67,60 @@ public function setPageFactory(PageFactory $pageFactory)
...
+            $headers = $session->getPage()->findAll('css', sprintf('#grid-%s table.grid thead th', $gridName));
```
Comment (juliensnz) [inline features/Context/DataGridContext.php]: please use the get grid which @jjanvier and @phaseinducer did today

File: spec/Pim/Bundle/UIBundle/Twig/StyleExtensionSpec.php [hunk 3]
```diff
@@ -28,6 +28,6 @@ function it_defines_filters()
 
     function it_highlights()
     {
-        $this->highlight('toto')->shouldReturn('<span class="highlight">toto</span>');
+        $this->highlight('toto')->shouldReturn('<span class="bold-purple">toto</span>');
```
Comment (juliensnz) [inline spec/Pim/Bundle/UIBundle/Twig/StyleExtensionSpec.php]: we should use highlight. I know that you are not responsible of that but `bold-purple` is really an awful css name :/

File: src/Pim/Bundle/UIBundle/Resources/public/css/pim.less [hunk 4]
```diff
@@ -1749,8 +1753,3 @@ h3.tab-header {
 .scopable-input .note-editor {
   display: inline-block;
 }
-
-.highlight {
```
Comment (juliensnz) [inline src/Pim/Bundle/UIBundle/Resources/public/css/pim.less]: this was actually the good solution :/

File: src/Pim/Bundle/UIBundle/Twig/StyleExtension.php [hunk 5]
```diff
@@ -30,7 +30,7 @@ public function getFilters()
      */
     public function highlight($content)
     {
-        return sprintf('<span class="highlight">%s</span>', $content);
+        return sprintf('<span class="bold-purple">%s</span>', $content);
```
Comment (juliensnz) [inline src/Pim/Bundle/UIBundle/Twig/StyleExtension.php]: same here

**Comment by juliensnz** [comment]
GTM

**Comment by grena** [comment]
GTM

**Pull request merged**